jobs/statistics/ArchiveUsageStatsLogFile.php

```diff
@@ -0,0 +1,80 @@
+<?php
+
+/**
+ * @file jobs/statistics/ArchiveUsageStatsLogFile.php
+ *
+ * Copyright (c) 2024 Simon Fraser University
+ * Copyright (c) 2024 John Willinsky
+ * Distributed under the GNU GPL v3. For full terms see the file docs/COPYING.
+ *
+ * @class ArchiveUsageStatsLogFile
+ *
+ * @ingroup jobs
+ *
+ * @brief Archive usage stats log file.
+ */
+
+namespace PKP\jobs\statistics;
+
+use APP\statistics\StatisticsHelper;
+use Exception;
+use PKP\file\FileManager;
+use PKP\job\exceptions\JobException;
+use PKP\jobs\BaseJob;
+use PKP\site\Site;
+use PKP\task\FileLoader;
+
+class ArchiveUsageStatsLogFile extends BaseJob
+{
+    /**
+     * The number of times the job may be attempted.
+     */
+    public $tries = 1;
```

Comment:
Shouldn't we try again in a couple of days and/or warn the sysadmin if there's something not working?

Reply:
About the retry, as we have files waiting to be processed, they are enough to trigger the retry 😁

Reply:
I believe it is OK to have tries = 1. If the job fails the message will be visible in the failed_jobs table, and then the admin can decide if it can be re-run or if the problem first needs to be solved.

Reply:
FYI, there's a setting to delete failed jobs here (which I think should be "off" by default!), so they will be lost after some time:
https://github.com/pkp/ojs/blob/f60b51191ded50462eb76a75d1c9c48882365640/config.TEMPLATE.inc.php#L560